app/router.js

```diff
@@ -58,6 +58,16 @@ Router.map(function() {
   this.route('school', { path: 'schools/:school_id'});
   this.route('assign-students', {path: '/admin/assignstudents'});
   this.route('myprofile');
+  this.route('course-rollover');
+  this.route('curriculumInventoryReports', {
```

Comment:
If you surround this in a if block to check if the feature is enabled will that work to prevent anyone with the URL from visiting the routes?

`if (config.IliosFeatures.curriculumInventory ) {`